impl/src/test/java/org/ehcache/impl/internal/spi/copy/DefaultCopyProviderTest.java

```diff
@@ -84,6 +90,39 @@ public void testCreateValueCopierWithSerializer() {
     assertThat(copyProvider.createValueCopier(Long.class, mock(Serializer.class), config), instanceOf(SerializingCopier.class));
   }
 
+  @Test
```

Comment:
This misses a test that close in well invoked on an instantiated copier type that implements `Closeable`